### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by franfranfranfran** [NJANGY/prj-rev-bwfs-dasmoto#5]
**GRADE**: Needs Improvement

**SUMMARY**: Good start on this project. With a few tweaks, the page will be looking great and you've done a nice job organizing the project folder by using a separate resources folder. Good use of classes to help label certain elements on the page as well!

Behind the scenes, there's just a couple things to keep in mind to make sure everything is perfect. Avoid repeating styling for similarly styled elements whenever possible. Also, make sure to keep your indentation as consistent as possible throughout both the html and css. Clean, concise code is crucial and only becomes more so as your projects grow in size and complexity.

As a next step, try using divs to reinforce the content structure of the page. Sectioning off each of the main content blocks will help with styling, readability, and creating a well-defined structure for the page.

Have fun and keep up the good work!

**Comment by NJANGY** [comment]
Thanks for the review.
I am the kind of person that needs to have the solution to understand what I did wrong. 
As it's new, I am already lost in all your comments- too many info without the concret image behind.
I understand for the pictures that indeed it has to be under my hands, here it was an exercice, but I take the note for future exercices or projects. 
when I go to my https://njangy.github.io/dasmoto/Dasmoto%20Arts/index.html - I don't see things as it was set up on my computer, probably missing a link somewhere.
To summarise, of course I need improvement, but for that I need more exercice like this one, and I didn't find this on the codeacademy, any possibilities to have more ? I need to practice to understand and capture things, doing 1 time is not enough :(  
thanks for having taking the time to review and comments my work.
Have a nice day
angelique
